src/main/java/net/neoforged/neoforge/client/event/RenderPlayerEvent.java

```diff
@@ -27,19 +27,19 @@
  */
 public abstract class RenderPlayerEvent extends RenderLivingEvent<AbstractClientPlayer, PlayerRenderState, PlayerModel> {
     @ApiStatus.Internal
-    protected RenderPlayerEvent(PlayerRenderState renderState, PlayerRenderer renderer, float partialTick, PoseStack poseStack, MultiBufferSource multiBufferSource, int packedLight) {
+    protected RenderPlayerEvent(PlayerRenderState renderState, LivingEntityRenderer<AbstractClientPlayer, PlayerRenderState, PlayerModel> renderer, float partialTick, PoseStack poseStack, MultiBufferSource multiBufferSource, int packedLight) {
```

Comment:
What's the thought behind loosening this from `PlayerRenderer` to `LivingRenderer`?

Reply:
Mostly just that the class doesn't have a hard restriction on it. If player's override the player renderer and extend the player model, they'd be able to reuse the event. If Neo does want to strictly require the `PlayerRenderer`, then I believe the `getRenderer` method should be overridden to provide specifically `PlayerRenderer` rather than `LivingEntityRenderer<AbstractClientPlayer, PlayerRenderState, PlayerModel>`

Reply:
I think the second way is more desired. Could you undo this change?

Reply:
Dang looks like linter got you. Cursed linters!